docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     environment:
       SPA_PATH: /openmrs/spa
       API_URL: /openmrs
-      SPA_CONFIG_URLS:
+      SPA_CONFIG_URLS: config-core_demo.json
```

Comment:
```suggestion
      SPA_CONFIG_URLS: /openmrs/spa/config.json
```

Reply:
Note I'm also renaming the file. I don't know that we need the same naming scheme here.

Reply:
Isn't this file already relative to /openmrs/spa/ by default @ibacher ?  Can't we just point this to "config.json" then?